acra/configuring-maintaining/general-configuration/acra-authmanager.md

```diff
@@ -0,0 +1,131 @@
+---
+title: acra-authmanager
```

Comment:
In some places (inside  acra/configuring-maintaining/general-configuration/*.md) we use `AcraFoo`, in others — `acra-foo`.
Just a note that we'll have to end up with a single variant.